.env.prod

```diff
@@ -1,10 +1,10 @@
 TESTFAIRY_ENABLED=0
 NOTIFICATIONS_WIP_ENABLED=1
-DEBUG_LOGS_ENABLED=0
 STUB_STATUS_GO=0
 ETHEREUM_DEV_CLUSTER=0
 MAINNET_NETWORKS_ENABLED=0
 ERC20_ENABLED=0
 OFFLINE_INBOX_ENABLED=0
 LOG_LEVEL=info
+LOG_LEVEL_STATUS_GO=
```

Comment:
Missed this in review, but noticed now. Seems wrong? @PombeirP

Reply:
@oskarth That is based on this suggestion: https://github.com/status-im/status-react/pull/2803#issuecomment-354145328

Reply:
Oh my bad, missed this. Gotcha.